testdata/project-v3-addon/config/default/manager_auth_proxy_patch.yaml

```diff
@@ -21,5 +21,6 @@ spec:
           name: https
       - name: manager
         args:
```

Comment:
we might need to setup the auth proxy to allow that ( - -skip-auth-regex=^/readyz )